src/evaluations/data/evaluation_configs.py

```diff
@@ -153,24 +158,25 @@ def _frequency_smoke_test(num_runs=NUM_RUNS_VALUE):
           ScenarioConfig(
               name='homogeneous',
               set_generator_factory=(
-                frequency_set_generator.HomogeneousMultiSetGenerator.
-                get_generator_factory_with_num_and_size(
-                    universe_size=int(2e5), num_sets=10, set_size=int(2e4),
-                    freq_rates=[1]*10, freq_cap=3))),
+                  frequency_set_generator.HomogeneousMultiSetGenerator
+                  .get_generator_factory_with_num_and_size(
+                      universe_size=universe_size, num_sets=10,
+                      set_size=int(2e4), freq_rates=[1]*10, freq_cap=3))),
```

Comment:
doing this list multiplication trick has bitten me in the butt before but you probably won't get in trouble since you won't be modifying the list?  if I remember correctly the issue has to do with getting 10 copies of the same object instead of 10 new objects, so also with integers it's less of a problem.  That being said, in general ever since then I  just do a list comprehension instead [1 for _ in range(10)] so there's no chance of getting bitten.  This is just an FYI, no need to fix it here unless you want to

Reply:
Acknowledged. I will not change in this PR. I will follow list comprehension in the future PRs. Thanks!